packages/ui/test/working-groups/components/MyEarningsStat.test.tsx

```diff
@@ -0,0 +1,110 @@
+import { cryptoWaitReady } from '@polkadot/util-crypto'
+import { screen, render, waitForElementToBeRemoved } from '@testing-library/react'
+import { startOfToday, subDays } from 'date-fns'
+import React from 'react'
+import { BrowserRouter } from 'react-router-dom'
```

Comment:
Dunno why I was using BrowserRouter in tests as there's MemoryRouter as well. I'll fix that